packages/metacoin/compiler.json

```diff
@@ -6,6 +6,7 @@
             "*": {
                 "*": [
                     "abi",
+                    "devdoc",
```

Comment:
This file changed, but there is no corresponding change to any artifacts anywhere.  Seems like there should be either both or neither.

Reply:
This is an example contract. I'll add an example devdoc to one of the methods. But it's not going to be re-exported in `contract-artifacts`. I think we should keep this to show that we include devdoc as a best practice, since we expect `abi-gen` to depend on devdocs.

Reply:
Makes sense, thanks.  (Micro nit: `abi-gen` should _support_ devdocs, but not _depend on_ them.  Some users will have _no_ devdocs, and that's okay.)